### ISSUE TITLE
Won’t do anything 

### ISSUE THREAD

**Issue by casey321123** [Genwald/Data-Arc-Dumper#1]
EmuMMC 10.0.0 atmosphere 0.12.0
No matter what option I select upon opening the application does nothing. The screen doesn’t change, just shows the options indefinitely regardless of input

**Comment by Genwald** [comment]
Yeah, this needs to be updated to use a newer version of libnx due to a system update that broke button input in homebrew. You can still use it by setting "enable_deprecated_hid_mitm = u8!0x1" in /atmosphere/config/system_settings.ini as a workaround that atmosphere implemented for this issue.
But as the readme says "Project continued here: [Ultimate Mod Manager](https://github.com/ultimate-research/UltimateModManager)". Ultimate Mod Manager includes the entirety of this program as well as many updates to it, please use that instead.
